react/Rating/Rating.js

```diff
@@ -33,6 +33,7 @@ const getStar = (
 };
 
 const Rating = ({
+  ratingComment,
```

Comment:
I feel `ratingComment` is superfluous considering we are in the context of `<Rating>` how about just `comment`?